addon/plugins/list/nodes/list-nodes.ts

```diff
@@ -319,6 +323,8 @@ function renderEntry(entry: ListPathEntry) {
   const { style, pos } = entry;
   if (style === 'lower-alpha') {
     return indexToAlpha(pos);
+  } else if (style === 'upper-alpha') {
+    return indexToAlpha(pos).toUpperCase();
```

Comment:
This seems the most simple way to accomplish this, but if extending `indexToAlpha` a bit to support upper case fits better here, just let me know.

Reply:
this solution looks good to me!